.github/workflows/release.yml

```diff
@@ -0,0 +1,130 @@
+name: 'CD'
+
+on:
+  workflow_call:
+    inputs:
+      flavor:
+        description: 'The release flavor'
+        type: 'string'
+        required: true
+      solution:
```

Comment:
Can this be a list?

I understand the list cannot be dynamic. But I guess, it won't change too often aswell.